src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,69 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+import duke.parse.DateTimeManager;
+
+/**
+ * Represents a deadline.
+ * A deadline contains an end time.
+ */
+public class Deadline extends Task {
+    private LocalDateTime deadline;
```

Comment:
Perhaps would it be better to name this variable to another name as it is same as the Deadline class and readers might get confused?